src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random rand = new Random();
+
+    public Random getRand() {
+        return rand;
+    }
+
+    public void setRand(Random rand) {
+        this.rand = rand;
+    }
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), rand.nextInt(100));
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. Here, `new ColorSupplier()` creates a new `ColorSupplier` instance on every method call. It's better to create a single `ColorSupplier` instance and reuse it.